packages/sampler/.storybook/preview.js

```diff
@@ -1,6 +1,7 @@
 import {configureActions} from '@enact/storybook-utils/addons/actions';
 import {getBooleanType, getObjectType} from '@enact/storybook-utils/addons/controls';
-import {DocsPage, DocsContainer} from '@enact/storybook-utils/addons/docs';
+import {DocsContainer, Primary, Title} from '@enact/storybook-utils/addons/docs';
```

Comment:
We need to update storybook-utils version to make this work.
We'll release storybook-utils soon so let's update the PR after release.

Reply:
We've released storybook-utils 5.0.1. Please update the PRs.